### PR TITLE
add schedule selection to pkg_edit.php

### PR DESCRIPTION
**Pull request by xbipin**
this allows schedule selection to pkg_edit.php, this can then be
utilized by packages to set schedules in many other places. This is the
first set of changes, will further modify other files to make full use
of schedules.

### CODE REVIEW

File: usr/local/www/pkg_edit.php [hunk 1]
```diff
@@ -808,6 +808,33 @@ function enablechange() {
...
+			case "schedule_selection":
+				$input = "<select id='{$pkga['fieldname']}' name='{$pkga['fieldname']}'>\n";
+				$schedules = array();
+				$schedules[] = "none";
+				if(is_array($config['schedules']['schedule'])) {
+					foreach ($config['schedules']['schedule'] as $schedule) {
+						if ($schedule['name'] <> "")
+							$schedules[] = $schedule['name'];
+						}
+					}
```
Comment (rbgarga) [inline usr/local/www/pkg_edit.php]: Looks like it's wrong indented here

File: usr/local/www/pkg_edit.php [hunk 2]
```diff
@@ -808,6 +808,33 @@ function enablechange() {
...
+				foreach($schedules as $schedule) {
+					$selected = ($value == $schedule ? "selected=\"selected\"" : "");
+					if($schedule == "none")
+						$input .= "<option value=\"\" {$selected}>{$schedule}</option>\n";
+					else
+						$input .= "<option value=\"{$schedule}\" {$selected}>{$schedule}</option>\n";
+					}
```
Comment (rbgarga) [inline usr/local/www/pkg_edit.php]: And also here

File: usr/local/www/pkg_edit.php [hunk 3]
```diff
@@ -808,6 +808,33 @@ function enablechange() {
...
+				$input .= "</select>\n<br />\n" . fixup_string($pkga['description']) . "\n";
+				if(isset($pkga['advancedfield']) && isset($adv_filed_count)) {
+					$js_array[] = $pkga['fieldname'];
+					$advanced .= display_advanced_field($pkga['fieldname']).$input;
+					$advanced .= "</div>\n";
+					}
```
Comment (rbgarga) [inline usr/local/www/pkg_edit.php]: And also here

**Comment by xbipin** [comment]
actually i knew it was wrong but rest of the file was such so i though to follow that only

**Comment by xbipin** [comment]
fixed it

**Comment by xbipin** [comment]
can u check this and commit if all good?

**Comment by xbipin** [comment]
bump
also kindly let me know what files do i need to look at to trigger services on and off, for a start i would like to use this schedule thing in upnp and then later create something like a framework where other packages can also use schedules to turn things off and on

**Pull request merged**